src/components/tabs/tabs.spec.js

```diff
@@ -571,364 +571,443 @@ describe("Tabs", () => {
-            errorMessage=""
-            warningMessage=""
-            infoMessage=""
-            title="Tab Title 1"
-            tabId="uniqueid1"
-          >
-            <div />
-          </Tab>
-        </Tabs>
-      ).find(Tab);
+  describe("when in Drawer sidebar", () => {
+    let container;
+    let wrapper;
+
+    beforeEach(() => {
+      container = document.createElement("div");
+      container.id = "enzymeContainer";
+      document.body.appendChild(container);
+
+      wrapper = mount(
+        <DrawerSidebarContext.Provider value={{ isInSidebar: true }}>
+          <Tabs>
+            <Tab
+              title="Tab Title 1"
+              tabId="uniqueid1"
+              errorMessage=""
+              warningMessage=""
+              infoMessage=""
+            />
+            <Tab
+              title="Tab Title 2"
+              tabId="uniqueid2"
+              errorMessage=""
+              warningMessage=""
+              infoMessage=""
+            />
+            <Tab
+              title="Tab Title 3"
+              tabId="uniqueid3"
+              errorMessage=""
+              warningMessage=""
+              infoMessage=""
+            />
+          </Tabs>
+        </DrawerSidebarContext.Provider>,
+        {
+          attachTo: document.getElementById("enzymeContainer"),
+        }
+      );
+    });
+
+    afterEach(() => {
+      if (container?.parentNode) {
+        container.parentNode.removeChild(container);
+      }
 
-      expect(tab.props().isTabSelected).toEqual(true);
-      expect(tab.props().title).toEqual("Tab Title 1");
-      expect(tab.props().tabId).toEqual("uniqueid1");
+      container = null;
+
+      wrapper?.unmount();
     });
+
+    it.each([0, 1, 2])(
+      "updates to make the next Tab visible when the down key is pressed",
+      (index) => {
+        act(() => {
+          simulate.keydown.pressArrowDown(wrapper.find(TabTitle).at(index));
+        });
+        wrapper.update();
+        const newIndex = index === 2 ? 0 : index + 1;
+
+        expect(
+          wrapper.find(StyledTabTitle).at(newIndex).getDOMNode()
+        ).toBeFocused();
+      }
+    );
+
+    it.each([0, 1, 2])(
+      "updates to make the next Tab visible when the up key is pressed",
+      (index) => {
+        act(() => {
+          simulate.keydown.pressArrowUp(wrapper.find(TabTitle).at(index));
+        });
+        wrapper.update();
+        const newIndex = index === 0 ? 2 : index - 1;
+
+        expect(
+          wrapper.find(StyledTabTitle).at(newIndex).getDOMNode()
+        ).toBeFocused();
+      }
+    );
   });
+});
 
-  describe("Validation", () => {
-    const updateProps = (wrapper, props) => {
-      wrapper.setProps({
-        errors: { ...wrapper.props().errors, ...props.errors },
-        warnings: { ...wrapper.props().warnings, ...props.warnings },
-        infos: { ...wrapper.props().infos, ...props.infos },
-      });
-      wrapper.update();
-    };
+describe("With one Tab", () => {
```

Comment:
Could you place the "describe" with the correct space here please, just to keep the format.